src/System.IO.FileSystem/tests/DirectoryInfo/ToString.cs

```diff
@@ -45,5 +45,34 @@ public void DriveOnlyReturnsDrive_Windows()
             var info = new DirectoryInfo(path);
             Assert.Equal(path, info.ToString());
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsFullFramework))]
+        public void ParentToString_Framework()
+        {
+            ParentToString(false);
+        }
```

Comment:
I think we've been removing such framework-specific tests from the repo, which is why Viktor removed that test configuration.  I suggest just writing one test that validates the expected core behavior.